docker-compose.yml

```diff
@@ -90,7 +90,6 @@ services:
       context: .
       dockerfile: Dockerfile
     volumes:
-      - ./logs:/bot/logs
```

Comment:
Should this remain here even if its optional?

Reply:
You've commented on the diff that removes it.